test/spec/features/service/start_spec.rb

```diff
@@ -18,7 +18,7 @@
     expect(k.run).to be_truthy
     sleep 1
     k = run("kontena service show test-1")
-    expect(k.out.scan('state: running').size).to eq(1)
+    expect(k.out.scan('desired_state: running').size).to eq(1)
```

Comment:
If these were `expect(k.out).to match /desired_state: running/` it would be easier to diagnose spec failures, because it would log the output if it didn't match.